### ISSUE TITLE
Like Button: Change behavior when third-party cookies are blocked

### ISSUE THREAD

**Issue by AtrumGeost** [Automattic/wp-calypso#47837]
#### Steps to reproduce

1. Block third-party cookies on your browser of preference
2. Log in to WordPress.com
3. Access a page or post that has the Like button
4. Try giving a like to a post or page

#### What I expected

Since third party cookies are blocked, the best would be to ask the visitor to log into their WordPress.com account to properly like the page or post

#### What happened instead

It looks like there's an attempt to bring up the login window, and it is instantly closed. Also, it looks like the Like worked, but in fact, it didn't

#### Browser / OS version

Tested on macOS Big Sur 11.0.1 (20B29)

Using Firefox 83.0 (64-bit) — The issue was initially reported for Safari

#### Screenshot / Video

![Screen Capture on 2020-11-27 at 11-44-05](https://user-images.githubusercontent.com/3696121/100470617-93f5dc80-30a6-11eb-8da3-537a005a4d30.gif)

#### Context / Source

#user-report

25942932-hc (the user asked for a follow-up when this is fixed)

**Comment by cathymcbride** [comment]
Hi @AtrumGeost! In your repro steps, step #2 is to log in, but you expect to be prompted to log in when you like something? Should step #2 be log *out*? Also, is there anything in the console of interest?

**Comment by AtrumGeost** [comment]
Hi @cathymcbride !

> but you expect to be prompted to log in when you like something? Should step #2 be log out?

Correct. That's because people who have third-party cookies enabled think that if they are already logged in they'll be able to like a post. That step is to demonstrate that even if you are already logged into your WordPress.com account, you are not able to like a post.

On the other hand, if you are already logged out, you'll see the proper log-in window. But it won't allow me to like the post either. So maybe showing the login window is not a solution, but any clues to WordPress.com users that their likes won't be saved if they have third-party cookies blocked.


>Also, is there anything in the console of interest?

Yup, there are some warnings about `Cookie “” has been rejected as third-party.` Here's a video:

https://d.pr/v/zY59Ah/ZtciD5Sgb4

**Comment by cathymcbride** [comment]
OK! Thank you!

**Comment by aneeshd16** [comment]
I have tested this in Chrome, Safari and Firefox (with 3rd party cookies disabled), and I could not reproduce the bug. In Firefox, the popup is blocked; to show it, the user has to click on "Preferences" and then "Show ..."
![image](https://user-images.githubusercontent.com/5436027/118476283-ee552900-b72a-11eb-93c9-c71c0fcfc24b.png)

This was probably fixed earlier this year, I found some context in p5TWut-ub-p2. I haven't been able to find the exact date yet, but I'll circle back when I do! Can we close this issue?
